### PR TITLE
[Update] WAAPI Post

### PR DESCRIPTION
**Pull request by una**
In response to a valid point raised on [Twitter](https://twitter.com/brianskold/status/1295943386092826625) on the perception of this article, I've updated the title and copy to better reflect the state of the web community.

Changes proposed in this pull request:

- Change title to "Using the full power of the Web Animations API"
- Update paragraph to show primary/immediate adoption from Mozilla and Safari
- Update caption to specify Chromium

### CODE REVIEW

**Review by birtles — APPROVED** [review]
Looks good to me.

A few extra suggestions:

* Line 181 is missing `` quotes around the template string.
* In the section about replaceable animations, "To address this problem, replaceable animations were introduced in Chromium 83" makes it sound like this is a Chromium invention. Could we say, "To address this problem, Web Animations' replaceable animations were implemented in..."
* A final call to action mentioning cross-browser support would be a good idea for a site about the Web platform. e.g. "With the exception of compositing modes which, at the time of writing, were not yet available in Safari, these features are also available in Firefox and Safari so please give them a try today."

(Technically Firefox has supported compositing modes for years but we didn't ship them since we were waiting for other browsers to catch up. Chromium caught us off guard by deciding to ship it without waiting for Safari to catch up so Firefox only turned it on for release channels recently in version 80. That ships next week, however, so it should be fine to treat it as shipped in Firefox.)

File: src/site/content/en/blog/web-animations/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Web Animations API improvements in Chromium 84
+title: Using the full power of the Web Animations API
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply src/site/content/en/blog/web-animations/index.md:2]: Even though I like the new title, I think she's right. The article is focused on Chrome 84.

File: src/site/content/en/blog/web-animations/index.md [hunk 2]
```diff
@@ -20,12 +20,12 @@ When used correctly, [animations improve user perception and memory](https://www
 
 The [Web Animations API](https://www.w3.org/TR/web-animations-1/) is a tool that enables developers to write [imperative animations with JavaScript](https://www.youtube.com/watch?v=WaNoqBAp8NI). It was written to underpin both CSS animation and transition implementations and enable future effects to be developed, as well as existing effects to be composed and timed.
 
-While it has a long history, originally launching in Chromium 36, the latest release, Chromium 84, brings us a slew of previously unsupported features. 
+While [Mozilla](https://birtles.github.io/areweanimatedyet/) and [Safari](https://webkit.org/status/#specification-web-animations), have already implemented the full set of spec [features](https://caniuse.com/#feat=web-animation) in their browsers, Chromium 84 brings a slew of previously unsupported features to [Chrome and Edge](https://developer.microsoft.com/en-us/microsoft-edge/status/webanimationsjavascriptapi/) enabling cross-browser interoperability.
```
Comment (jpmedley) [inline src/site/content/en/blog/web-animations/index.md:23]:
```suggestion
While [Mozilla](https://birtles.github.io/areweanimatedyet/) and [Safari](https://webkit.org/status/#specification-web-animations), has already implemented the full set of spec [features](https://caniuse.com/#feat=web-animation) in its browsers, Chromium 84 brings a slew of previously unsupported features to [Chrome and Edge](https://developer.microsoft.com/en-us/microsoft-edge/status/webanimationsjavascriptapi/) enabling cross-browser interoperability.
```

Comment (jpmedley) [reply src/site/content/en/blog/web-animations/index.md:23]: The Google standard is American English. I don't know why I didn't request this change before.

File: src/site/content/en/blog/web-animations/index.md [hunk 3]
```diff
@@ -20,12 +20,12 @@ When used correctly, [animations improve user perception and memory](https://www
 
 The [Web Animations API](https://www.w3.org/TR/web-animations-1/) is a tool that enables developers to write [imperative animations with JavaScript](https://www.youtube.com/watch?v=WaNoqBAp8NI). It was written to underpin both CSS animation and transition implementations and enable future effects to be developed, as well as existing effects to be composed and timed.
 
-While it has a long history, originally launching in Chromium 36, the latest release, Chromium 84, brings us a slew of previously unsupported features. 
+While [Mozilla](https://birtles.github.io/areweanimatedyet/) and [Safari](https://webkit.org/status/#specification-web-animations), has already implemented the full set of spec [features](https://caniuse.com/#feat=web-animation) in its browsers, Chromium 84 brings a slew of previously unsupported features to [Chrome and Edge](https://developer.microsoft.com/en-us/microsoft-edge/status/webanimationsjavascriptapi/) enabling cross-browser interoperability.
```
Comment (jpmedley) [inline src/site/content/en/blog/web-animations/index.md:23]:
```suggestion
While [Firefox](https://birtles.github.io/areweanimatedyet/) and [Safari](https://webkit.org/status/#specification-web-animations), have already implemented the full set of spec [features](https://caniuse.com/#feat=web-animation), Chromium 84 brings a slew of previously unsupported features to [Chrome and Edge](https://developer.microsoft.com/en-us/microsoft-edge/status/webanimationsjavascriptapi/) enabling cross-browser interoperability.
```

Comment (jpmedley) [reply src/site/content/en/blog/web-animations/index.md:23]: Also, can you please link to something official from Mozilla.

**Comment by birtles** [comment]
Thanks for doing this. Unfortunately GitHub doesn't allow comments on untouched lines so my other feedback might have been missed.

In case it's of use, the other feedback I had was:

> * Line 181 is missing `` quotes around the template string.
> * In the section about replaceable animations, "To address this problem, replaceable animations were introduced in Chromium 83" makes it sound like this is a Chromium invention. Could we say, "To address this problem, Web Animations' replaceable animations were implemented in..."
> * A final call to action mentioning cross-browser support would be a good idea for a site about the Web platform. e.g. "With the exception of compositing modes which, at the time of writing, were not yet available in Safari, these features are also available in Firefox and Safari so please give them a try today."

**Pull request merged**